app/controllers/expenses_additional_controller.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+class ExpensesAdditionalController < StepsController
+  ATTRIBUTES = %w[
+    monthly_care_expenses
+    monthly_medical_expenses
+    monthly_court_ordered_expenses
+    monthly_tax_deductible_expenses
+  ].freeze
+
+  ARRAY_ATTRIBUTES = {
+    care_expenses: %w[
+      childcare
+      elderly_care
+      disabled_adult_care
+    ],
+    medical_expenses: %w[
+      health_insurance
+      co_pays
+      prescriptions
+      dental
+      in_home_care
+      transportation
+      hospital_bills
+      other
+    ],
+    court_ordered_expenses: %w[
+      child_support
+      alimony
+    ],
+    tax_deductible_expenses: %w[
+      student_loan_interest
+      other_tax_deductible
+    ],
+  }.freeze
+
+  def edit
+    set_sections
+
+    attribute_array = ARRAY_ATTRIBUTES.keys.flat_map do |key|
```

Comment:
maybe this should happen in a well-named private method? there is enough happening in these controller methods that I am getting a but turned around while reading them